students/grzegorz_kowalik/lesson_02_flow_control/bishop_moves.py

```diff
@@ -0,0 +1,14 @@
+from math import fabs
+
+begin_row = int(input())
+begin_col = int(input())
+target_row = int(input())
+target_col = int(input())
+
+d_rows = begin_row - target_row
+d_cols = begin_col - target_col
+
+if fabs(d_rows) == fabs(d_cols):
```

Comment:
W pythonie są dwie funkcję zwracające wartość bezwzględną: abs i math.fabs()
fabs sprawdza czy może zrobić konwersje danych wejściowych do float, jako że nasze dane wejściowe są int, nie jest to koniecznie, dlatego lepszym wyborem jest tutaj abs.
Więcej informacji o różnicach:
https://stackoverflow.com/questions/10772302/python-abs-vs-fabs